server/protocols/webadmin/webadmin-mailbox/src/main/java/org/apache/james/webadmin/dto/QuotaDTO.java

```diff
@@ -102,5 +106,13 @@ public String toString() {
             .toString();
     }
 
+    @JsonIgnore
+    public boolean isValid() {
+        return ImmutableList.of(count, size)
+            .stream()
+            .filter(quota -> quota.map(QuotaValue::asLong).orElse(-1L) < -1)
```

Comment:
magic value?

Reply:
Also this nkn trival condition needs extraction i n a named method to add some contexte.